src/core/core.scale.js

```diff
@@ -317,6 +315,12 @@ var Scale = Element.extend({
 
 		me._ticks = ticks;
 
+		// _configure is called twice, once here, once from core.controller.
+		// Here we haven't been positioned yet, but dimensions are correct.
+		// Variables set in _configure are needed for calculateTickRotation, and
+		// its ok coordinates are not correct there, only dimensions matter
```

Comment:
The same thing is actually true of `setDimensions` which is something I spent a bit of time figuring out in the past couple days. Can we move `_configure` to happen next to `setDimensions` and make the comment apply to both?

nit: its ok -> it's ok that

Reply:
Comment updated. `_configure` cant be moved earlier, many scales need either data limits determined or ticks build to compute values.